### PR TITLE
[zh]sync setup-3

### PR DESCRIPTION
**Pull request by wlp1153468871**

### CODE REVIEW

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 1]
```diff
@@ -170,11 +269,21 @@ In order to use it, cgroup v2 must be supported by the CRI runtime as well.
...
+
+## CRI version support {#cri-versions}
+
+Your container runtime must support at least v1alpha2 of the container runtime interface.
+
+Kubernetes {{< skew currentVersion >}}  defaults to using v1 of the CRI API.If a container runtime does not support the v1 API, the kubelet falls back to using the (deprecated) v1alpha2 API instead.
 -->
-如果你想迁移到现有 kubeadm 托管集群中的 `systemd` cgroup 驱动程序，
-遵循此[迁移指南](/zh/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver)。
+如果你希望迁移到现有 kubeadm 托管集群中的 `systemd` cgroup 驱动程序，请按照 [配置 cgroup 驱动程序](/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver/)。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:280]:
```suggestion
如果你希望将现有 kubeadm 托管集群迁移到 `systemd` cgroup 驱动程序，
请按照[配置 cgroup 驱动程序](/zh/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver/)操作。
```

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 2]
```diff
@@ -170,11 +269,21 @@ In order to use it, cgroup v2 must be supported by the CRI runtime as well.
...
+
+## CRI 版本支持 {#cri-versions}
+
+你的容器运行时必须至少支持容器运行时接口的 v1alpha2。
+
+Kubernetes {{< skew currentVersion >}} 默认使用 v1 的 CRI API。如果容器运行时不支持 v1 API，则 kubelet 回退到使用（已弃用的）v1alpha2 API。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:286]:
```suggestion
Kubernetes {{< skew currentVersion >}} 默认使用 v1 的 CRI API。如果容器运行时不支持 v1 API，
则 kubelet 会回退到使用（已弃用的）v1alpha2 API。

```

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 3]
```diff
@@ -186,131 +295,40 @@ if you wish to migrate to the `systemd` cgroup driver in existing kubeadm manage
...
 
-安装和配置的先决条件：
+<!-- Follow the instructions for [getting started with containerd](https://github.com/containerd/containerd/blob/main/docs getting-started.md). Return to this step once you've created a valid configuration file, `config.toml`.  -->
 
-```shell
-cat <<EOF | sudo tee /etc/modules-load.d/containerd.conf
-overlay
-br_netfilter
-EOF
+按照[开始使用 containerd](https://github.com/containerd/containerd/blob/main/docs/getting-started.md) 的说明进行操作。 创建有效的配置文件 `config.toml` 后返回此步骤。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:309]:
```suggestion
按照[开始使用 containerd](https://github.com/containerd/containerd/blob/main/docs/getting-started.md) 的说明进行操作。
创建有效的配置文件 `config.toml` 后返回此步骤。
```

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 4]
```diff
@@ -345,352 +363,86 @@ When using kubeadm, manually configure the
...
-| Debian Testing   | `Debian_Testing`  |
+<!-- To install CRI-O, follow [CRI-O Install Instructions](https://github.com/cri-o/cri-o/blob/main/install.md#readme). -->
 
-<br />
-然后，将 `$VERSION` 设置为与你的 Kubernetes 相匹配的 CRI-O 版本。
-例如，如果你要安装 CRI-O 1.20, 请设置 `VERSION=1.20`.
-你也可以安装一个特定的发行版本。
-例如要安装 1.20.0 版本，设置 `VERSION=1.20.0:1.20.0`.
-<br />
+要安装 CRI-O，请按照 [CRI-O 安装说明](https://github.com/cri-o/cri-o/blob/main/install.md#readme)。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:371]:
```suggestion
要安装 CRI-O，请按照 [CRI-O 安装说明](https://github.com/cri-o/cri-o/blob/main/install.md#readme)执行操作。
```

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 5]
```diff
@@ -345,352 +363,86 @@ When using kubeadm, manually configure the
...
-curl -L https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:$VERSION/$OS/Release.key | sudo apt-key --keyring /etc/apt/trusted.gpg.d/libcontainers.gpg add -
-curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/Release.key | sudo apt-key --keyring /etc/apt/trusted.gpg.d/libcontainers.gpg add -
+<!-- CRI-O uses the systemd cgroup driver per default, which is likely to work fine for you. To switch to the `cgroupfs` cgroup driver, either edit `/etc/crio/crio.conf` or place a drop-in configuration in `/etc/crio/crio.conf.d/02-cgroup-manager.conf`, for example: -->
 
-sudo apt-get update
-sudo apt-get install cri-o cri-o-runc
-```
-
-{{% /tab %}}
+CRI-O 默认使用 systemd cgroup 驱动程序，这对你来说可能工作得很好。要切换到 `cgroupfs` cgroup 驱动程序，请编辑 `/etc/crio/crio.conf` 或在 `/etc/crio/crio.conf.d/02-cgroup-manager.conf` 中放置一个插入式配置 ，例如：
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:379]:
```suggestion
CRI-O 默认使用 systemd cgroup 驱动程序，这对你来说可能工作得很好。要切换到 `cgroupfs` cgroup 驱动程序，
请编辑 `/etc/crio/crio.conf` 或在 `/etc/crio/crio.conf.d/02-cgroup-manager.conf` 中放置一个 drop-in 配置 ，例如：
```

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 6]
```diff
@@ -345,352 +363,86 @@ When using kubeadm, manually configure the
...
+  -->
 
-{{% tab name="openSUSE Tumbleweed" %}}
-
-```shell
-sudo zypper install cri-o
-```
-{{% /tab %}}
-{{% tab name="Fedora" %}}
+你还应该注意更改的 `conmon_cgroup`，当使用 CRI-O 和 `cgroupfs` 时，必须将其设置为值 `pod`。 通常需要保持 kubelet 的 cgroup 驱动配置（通常通过 kubeadm 完成）和 CRI-O 同步。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:390]:
```suggestion
你还应该注意到 `conmon_cgroup` 被更改，当使用 CRI-O 和 `cgroupfs` 时，必须将其设置为值 `pod`。
通常需要保持 kubelet 的 cgroup 驱动配置（通常通过 kubeadm 完成）和 CRI-O 同步。
```

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 7]
```diff
@@ -345,352 +363,86 @@ When using kubeadm, manually configure the
...
 -->
-将 `$VERSION` 设置为与你的 Kubernetes 相匹配的 CRI-O 版本。
-例如，如果要安装 CRI-O 1.20，请设置 `VERSION=1.20`。
-你可以用下列命令查找可用的版本：
+对于 CRI-O，CRI 套接字默认为 `/var/run/crio/crio.sock`。
 
-```shell
-sudo dnf module list cri-o
-```
+### Docker 引擎 {#docker}
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:400]: Docker Engine 可视为专有名词，不要翻译

```suggestion
### Docker Engine {#docker}
```

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 8]
```diff
@@ -345,352 +363,86 @@ When using kubeadm, manually configure the
...
 
-CRI-O 不支持在 Fedora 上固定到特定的版本。
+<!-- {{< note >}}
+These instructions assume that you are using the [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) adapter to integrate
+Docker Engine with Kubernetes.
+{{< /note >}} -->
 
-然后执行
+提示：这些说明假设你使用 [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) 适配器来集成带有 Kubernetes 的 Docker 引擎。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:407]: 同上，保留短代码。

Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:407]:
```suggestion
以下操作假设你使用 [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) 适配器来集成将
Docker Engine 与 Kubernetes 集成。
```

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 9]
```diff
@@ -345,352 +363,86 @@ When using kubeadm, manually configure the
...
 
-```shell
-sudo dnf module enable cri-o:$VERSION
-sudo dnf install cri-o --now
-```
+<!-- 1. On each of your nodes, install Docker for your Linux distribution as per [Install Docker Engine](https://docs.docker.com/engine/install/#server). -->
 
-{{% /tab %}}
-{{< /tabs >}}
+1. 在你的每个节点上，按照 [安装 Docker 引擎](https://docs.docker.com/engine/install/#server) 为你的 Linux 发行版安装 Docker。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:411]:
```suggestion
1. 在你的每个节点上，遵循[安装 Docker 引擎](https://docs.docker.com/engine/install/#server)指南为你的
   Linux 发行版安装 Docker。
```

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 10]
```diff
@@ -345,352 +363,86 @@ When using kubeadm, manually configure the
...
-   -->
-   对于运行 Linux 内核版本 4.0 或更高版本，或使用 3.10.0-51 及更高版本的 RHEL
-   或 CentOS 的系统，`overlay2`是首选的存储驱动程序。
-   {{< /note >}}
-
-<!--
-3. Restart Docker and enable on boot:
--->
-3. 重新启动 Docker 并在启动时启用：
+除了容器运行时，你的集群还需要一个有效的 [网络插件](/docs/concepts/cluster-administration/networking/#how-to-implement-the-kubernetes-networking-model)。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:446]:
```suggestion
除了容器运行时，你的集群还需要有效的[网络插件](/zh/docs/concepts/cluster-administration/networking/#how-to-implement-the-kubernetes-networking-model)。
```

Comment (wlp1153468871) [reply content/zh/docs/setup/production-environment/container-runtimes.md]: @tengqm updated

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 11]
```diff
@@ -24,32 +26,130 @@ what is involved and describes related tasks for setting up nodes.
...
+
+If you are running a version of Kubernetes other than v{{< skew currentVersion >}}, check the documentation for that version.
 {{< /note >}}
+
+ -->
+ {{< note >}}
+提示：v1.24 之前的 Kubernetes 版本包括与 Docker Engine 的直接集成，使用名为 _dockershim_ 的组件。 
+这种特殊的直接整合不再是Kubernetes 的一部分
+（这次删除是[宣布](/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation)作为 v1.20 版本的一部分）。
+你可以阅读[检查 Dockershim 弃用是否会影响你](/zh/docs/tasks/administer-cluster/migrating-from-dockershim check-if-dockershim-deprecation-affects-you/) 以了解此删除可能会如何影响你。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:59]:
```suggestion
你可以阅读[检查 Dockershim 弃用是否会影响你](/zh/docs/tasks/administer-cluster/migrating-from-dockershim check-if-dockershim-deprecation-affects-you/)
以了解此删除可能会如何影响你。 
```

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 12]
```diff
@@ -24,32 +26,130 @@ what is involved and describes related tasks for setting up nodes.
...
  -->
-提示：对于其他操作系统，请查阅特定于你所使用平台的相关文档。
+
+## 安装和配置先决条件
+
+以下步骤将通用设置应用于 Linux 上的 Kubernetes 节点。
+
+如果你确定不需要某个特定设置，则可以跳过它。
+
+有关更多信息，请参阅 [网络插件要求](/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#network-plugin-requirements)
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:82]:
```suggestion
有关更多信息，请参阅[网络插件要求](/zh/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#network-plugin-requirements) 
```

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 13]
```diff
@@ -345,352 +383,101 @@ When using kubeadm, manually configure the
...
-deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/ /
-EOF
-cat <<EOF | sudo tee /etc/apt/sources.list.d/devel:kubic:libcontainers:stable:cri-o:$VERSION.list
-deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:/$VERSION/$OS/ /
-EOF
-
-curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/Release.key | sudo apt-key --keyring /etc/apt/trusted.gpg.d/libcontainers.gpg add -
-curl -L https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:$VERSION/$OS/Release.key | sudo apt-key --keyring /etc/apt/trusted.gpg.d/libcontainers-cri-o.gpg add -
+CRI-O 默认使用 systemd cgroup 驱动程序，这对你来说可能工作得很好。要切换到 `cgroupfs` cgroup 驱动程序，
+请编辑 `/etc/crio/crio.conf` 或在 `/etc/crio/crio.conf.d/02-cgroup-manager.conf` 中放置一个插入式配置 ，例如：
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:402]:
```suggestion
请编辑 `/etc/crio/crio.conf` 或在 `/etc/crio/crio.conf.d/02-cgroup-manager.conf` 中放置一个 drop-in 配置 ，例如：
```

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 14]
```diff
@@ -345,352 +383,101 @@ When using kubeadm, manually configure the
...
 -->
-启动 CRI-O：
 
-```shell
-sudo systemctl daemon-reload
-sudo systemctl enable crio --now
-```
+{{< note >}}
+以下操作假设你使用 [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) 适配器来集成将
+Docker Engine 与 Kubernetes 集成。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:436]:
```suggestion
以下操作假设你使用 [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd) 适配器来将
Docker Engine 与 Kubernetes 集成。
```

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 15]
```diff
@@ -345,352 +383,101 @@ When using kubeadm, manually configure the
...
-`pod` when using CRI-O with `cgroupfs`. It is generally necessary to keep the
-cgroup driver configuration of the kubelet (usually done via kubeadm) and CRI-O
-in sync.
- -->
-另请注意更改后的 `conmon_cgroup`，将 CRI-O 与 `cgroupfs` 一起使用时，
-必须将其设置为 `pod`。通常有必要保持 kubelet 的 cgroup 驱动程序配置
-（通常透过 kubeadm 完成）和 CRI-O 一致。
+2. Install [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd), following the instructions in that source code repository.
+-->
+2. 按照源代码存储库中的说明安装 [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd)。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:449]:
```suggestion
2. 按照源代码仓库中的说明安装 [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd)。
```

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 16]
```diff
@@ -345,352 +383,101 @@ When using kubeadm, manually configure the
...
+## {{% heading "whatsnext" %}}
 
-{{< note >}}
-<!--
-For more information refer to
-  - [Configure the Docker daemon](https://docs.docker.com/config/daemon/)
-  - [Control Docker with systemd](https://docs.docker.com/config/daemon/systemd/)
--->
-有关更多信息，请参阅
+As well as a container runtime, your cluster will need a working [network plugin](/docs/concepts/cluster-administration/networking/#how-to-implement-the-kubernetes-networking-model).
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:479]: 479 注释掉

**Comment by tengqm** [comment]
@wlp1153468871 Please help mark whether feedbacks are resolved after your revision.

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 17]
```diff
@@ -345,352 +384,105 @@ When using kubeadm, manually configure the
...
-{{< /tabs >}}
+1. 在你的每个节点上，遵循[安装 Docker 引擎](https://docs.docker.com/engine/install/#server)指南为你的
+   Linux 发行版安装 Docker。
 
 <!-- 
-Start CRI-O:
+2. Install [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd), following the instructions in that source code repository.
 -->
-启动 CRI-O：
+按照源代码仓库中的说明安装 [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd)。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:452]:
```suggestion
2. 按照源代码仓库中的说明安装 [`cri-dockerd`](https://github.com/Mirantis/cri-dockerd)。
```

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 18]
```diff
@@ -345,352 +384,105 @@ When using kubeadm, manually configure the
...
+ CRI-O uses the systemd cgroup driver per default, which is likely to work fine for you. To switch to the `cgroupfs` cgroup driver, either edit `/etc/crio/crio.conf` or place a drop-in configuration in `/etc/crio/crio.conf.d/02-cgroup-manager.conf`, for example: 
 -->
-在下列操作系统上安装 CRI-O, 使用下表中合适的值设置环境变量 `OS`:
 
-| 操作系统          | `$OS`             |
-| ---------------- | ----------------- |
-| Debian Unstable  | `Debian_Unstable` |
-| Debian Testing   | `Debian_Testing`  |
+CRI-O 默认使用 systemd cgroup 驱动程序，这对你来说可能工作得很好。要切换到 `cgroupfs` cgroup 驱动程序，
+Suggested change
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:403]: ?

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 19]
```diff
@@ -345,352 +384,105 @@ When using kubeadm, manually configure the
 <!--
 This section contains the necessary steps to install CRI-O as a container runtime.
 
-Use the following commands to install CRI-O on your system:
 -->
 本节包含安装 CRI-O 作为容器运行时的必要步骤。
 
-使用以下命令在系统中安装 CRI-O：
+<!-- To install CRI-O, follow [CRI-O Install Instructions](https://github.com/cri-o/cri-o/blob/main/install.md#readme). -->
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:390]:
```suggestion
<!--
To install CRI-O, follow [CRI-O Install Instructions](https://github.com/cri-o/cri-o/blob/main/install.md#readme).
-->
```

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 20]
```diff
@@ -186,131 +313,43 @@ if you wish to migrate to the `systemd` cgroup driver in existing kubeadm manage
...
 {{< /tabs >}}
 
 <!-- 
-#### Using the `systemd` cgroup driver {#containerd-systemd}
+On Linux the default CRI socket for containerd is `/run/containerd/containerd.sock`.
+On Windows the default CRI endpoint is `npipe://./pipe/containerd-containerd`.
+
+#### Configuring the `systemd` cgroup driver {#containerd-systemd}
 --> 
+在 Linux 上，containerd 的默认 CRI 套接字是`/run/containerd/containerd.sock`。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:349]:
```suggestion
在 Linux 上，containerd 的默认 CRI 套接字是 `/run/containerd/containerd.sock`。
```

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 21]
```diff
@@ -170,11 +285,23 @@ In order to use it, cgroup v2 must be supported by the CRI runtime as well.
...
+
+## CRI version support {#cri-versions}
+
+Your container runtime must support at least v1alpha2 of the container runtime interface.
+
+Kubernetes {{< skew currentVersion >}}  defaults to using v1 of the CRI API.If a container runtime does not support the v1 API, the kubelet falls back to using the (deprecated) v1alpha2 API instead.
 -->
-如果你想迁移到现有 kubeadm 托管集群中的 `systemd` cgroup 驱动程序，
-遵循此[迁移指南](/zh/docs/tasks/administer-cluster/kubeadm/configure-cgroup-driver)。
+如果你希望将现有 kubeadm 托管集群迁移到 `systemd` cgroup 驱动程序，
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:296]:
```suggestion
如果你希望将现有的由 kubeadm 管理的集群迁移到 `systemd` cgroup 驱动程序，
```

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 22]
```diff
@@ -24,32 +26,131 @@ what is involved and describes related tasks for setting up nodes.
...
+ {{< note >}}
+提示：v1.24 之前的 Kubernetes 版本包括与 Docker Engine 的直接集成，使用名为 _dockershim_ 的组件。 
+这种特殊的直接整合不再是Kubernetes 的一部分
+（这次删除是[宣布](/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation)作为 v1.20 版本的一部分）。
+你可以阅读[检查 Dockershim 弃用是否会影响你](/zh/docs/tasks/administer-cluster/migrating-from-dockershim check-if-dockershim-deprecation-affects-you/) 
+以了解此删除可能会如何影响你。 
+要了解如何使用 dockershim 进行迁移，请参阅[从 dockershim 迁移](/docs/tasks/administer-cluster/migrating-from-dockershim/)。
+
+如果你正在运行 v{{< skew currentVersion >}} 以外的 Kubernetes 版本，检查该版本的文档。
+{{< note >}}
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:64]:
```suggestion
{{< /note >}}
```

**Review by tengqm — COMMENTED** [review]
Please be professional and careful when working on revisions.

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 23]
```diff
@@ -24,32 +26,131 @@ what is involved and describes related tasks for setting up nodes.
...
+To load it explicitly, run `sudo modprobe br_netfilter`.
+
+In order for a Linux node's iptables to correctly view bridged traffic, verify that `net.bridge.bridge-nf-call-iptables` is set to 1 in your `sysctl` config. For example: 
+-->
+
+### 转发 IPv4 并让 iptables 看到桥接流量
+
+通过运行 `lsmod | grep br_netfilter` 来验证 `br_netfilter` 模块是否已加载。
+
+若要显式加载此模块，请运行`sudo modprobe br_netfilter`。
```
Comment (Sea-n) [inline content/zh/docs/setup/production-environment/container-runtimes.md:100]: nit: Missing space before `` `code` ``.

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 24]
```diff
@@ -325,9 +364,9 @@ To use the `systemd` cgroup driver in `/etc/containerd/config.toml` with `runc`,
 ```
 
 <!--
-If you apply this change make sure to restart containerd again:
+If you apply this change， make sure to restart containerd:
 -->
-如果您应用此更改，请确保再次重新启动 containerd：
+如果你应用此更改，请确保重新启动 containerd：
```
Comment (Sea-n) [inline content/zh/docs/setup/production-environment/container-runtimes.md:369]: nit: You've changed the `, ` to `，` in English comment.

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 25]
```diff
@@ -345,352 +384,106 @@ When using kubeadm, manually configure the
...
 <!--
-3. Restart Docker and enable on boot:
+To learn more about how to install Mirantis Container Runtime, visit [MCR Deployment Guide](https://docs.mirantis.com/mcr/20.10/install.html). 
 -->
-3. 重新启动 Docker 并在启动时启用：
+要了解有关如何安装 Mirantis Container Runtime 的更多信息，请访问 [MCR 部署指南](https://docs.mirantis.com/mcr/20.10/install.html)。
+<!-- 
+Check the systemd unit named `cri-docker.socket` to find out the path to the CRI socket.
+-->
+检查名为 cri-docker.socket 的 systemd 单元以找出 CRI 套接字的路径。
```
Comment (Sea-n) [inline content/zh/docs/setup/production-environment/container-runtimes.md:479]: nit: Missing `` `code` `` syntax.

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 26]
```diff
@@ -24,32 +26,131 @@ what is involved and describes related tasks for setting up nodes.
...
 {{< /note >}}
+
+ -->
+ {{< note >}}
+提示：v1.24 之前的 Kubernetes 版本包括与 Docker Engine 的直接集成，使用名为 _dockershim_ 的组件。 
+这种特殊的直接整合不再是Kubernetes 的一部分
+（这次删除是[宣布](/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation)作为 v1.20 版本的一部分）。
+你可以阅读[检查 Dockershim 弃用是否会影响你](/zh/docs/tasks/administer-cluster/migrating-from-dockershim check-if-dockershim-deprecation-affects-you/) 
+以了解此删除可能会如何影响你。 
+要了解如何使用 dockershim 进行迁移，请参阅[从 dockershim 迁移](/docs/tasks/administer-cluster/migrating-from-dockershim/)。
```
Comment (Sea-n) [inline content/zh/docs/setup/production-environment/container-runtimes.md:61]: nit: This article have Mandarin version.

**Review by Sea-n — COMMENTED** [review]
Here is some optional comments, thanks for your translation!

**Comment by tengqm** [comment]
@wlp1153468871 Are you ready to move this PR forward?

**Comment by tengqm** [comment]
/close
This one has been stagnated for a few days. I'm taking over it to kick it in.

**Pull request closed without merging**